### ISSUE TITLE
Break layout without checkbox

### ISSUE THREAD

**Issue by mex3890** [outl1ne/nova-sortable#164]
Without checkbox:
![image](https://user-images.githubusercontent.com/90221921/217538985-9b775707-53f6-4c04-bd08-c7963146005f.png)

With checkbox:
![image](https://user-images.githubusercontent.com/90221921/217539106-cc706ae4-18eb-44af-b470-fe50f0d47711.png)

**Comment by fabiobap** [comment]
Yeah same problem here, it displays fine with the checkbox and messes up without it

**Comment by thbighead** [comment]
Same here, it works fine on Nova 3 but crashes when using Nova 4

**Comment by mex3890** [comment]
Working on optimistdigital/nova-sortable version 3.4.2

**Comment by Tarpsvo** [comment]
I can't reproduce this.

![image](https://user-images.githubusercontent.com/2018660/225930133-1d748404-2c35-404c-9d80-2917ef928698.png)

Is this still an issue?

**Comment by mabdullahsari** [comment]
> Is this still an issue?

It is. `v3.4.3`

![image](https://github.com/outl1ne/nova-sortable/assets/24608797/5281b55a-86c1-4df0-bcbf-229883cedc36)

**Comment by cereneto** [comment]
> Is this still an issue?

It is for me, but only in has many list view. Index view work fine.
